CMakeLists.txt

```diff
@@ -120,7 +120,7 @@ endif()
 find_package(nudb REQUIRED)
 find_package(date REQUIRED)
 find_package(xxHash REQUIRED)
-find_package(wamr REQUIRED)
+find_package(wasm-xrplf REQUIRED)
```

Comment:
I'm concerned about this name given that we're potentially going to switch the runtime, thus making the name confusing. Would we use the same name if we switched the runtime? Why not call this `wamr-xrplf` instead?